neat_core.c

```diff
@@ -547,14 +576,25 @@ static void do_accept(neat_ctx *ctx, neat_flow *flow)
 
     newFlow->handle = (uv_poll_t *) malloc(sizeof(uv_poll_t));
     assert(newFlow->handle != NULL);
+
+#if defined(USRSCTP_SUPPORT)
+    newFlow->sock = newFlow->acceptfx(ctx, newFlow, flow->sock);
+    if (!newFlow->sock) {
+#else
```

Comment:
Same comment as above: Does you code handle TCP, UDP and UDPLite kernel sockets when using usrsctp?